.github/workflows/ci.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        version: [citadel, edifice]
```

Comment:
The README says we support citadel, edifice and fortress with Galactic, so I think it would be good to test them all here.

Note that since Galactic + Citadel and Galactic + Fortress aren't official combinations, `ros_ign` will also need to be built from source for them.